content/docs/user-guide/data-pipelines/defining-pipelines.md

```diff
@@ -0,0 +1,221 @@
+# Defining Pipelines
+
+DVC pipelines are written as collections of data processing <abbr>stages</abbr>
+in `dvc.yaml` files. These represent data workflows that you want to
+**reproduce** reliably later. Here's a sample 3-stage structure (in alphabetical
+order):
+
+```yaml
+stages:
+  extract: ... # stage 1 definition
+  load: ... # stage 3 definition
+  transform: ... # stage 2 definition
```

Comment:
let's make it classical ML `process ... featurize ... train ... evaluate`, not ETL/ELT

Reply:
> make it classical ML

At some point we discussed not making this guide ML-centric since pipelines are more general than that. But sure, I don't have a strong opinion, changed.